student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super(20)
+    @classroom = classroom
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice if you add `age, classroom, name = "Unknown", parent_permission = true`  to your `def` 🙏🏼 

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```